src/main/java/net/fabricmc/loom/configuration/providers/minecraft/SplitMinecraftProvider.java

```diff
@@ -45,20 +44,20 @@ public SplitMinecraftProvider(Project project) {
 	protected void initFiles() {
 		super.initFiles();
 
-		minecraftClientOnlyJar = file("minecraft-client-only.jar");
-		minecraftCommonJar = file("minecraft-common.jar");
+		minecraftClientOnlyJar = path("minecraft-client-only.jar");
+		minecraftCommonJar = path("minecraft-common.jar");
 	}
 
 	@Override
 	public List<Path> getMinecraftJars() {
-		return List.of(minecraftClientOnlyJar.toPath(), minecraftCommonJar.toPath());
+		return List.of(minecraftClientOnlyJar, minecraftCommonJar);
 	}
 
 	@Override
 	public void provide() throws Exception {
 		super.provide();
 
-		boolean requiresRefresh = isRefreshDeps() || !minecraftClientOnlyJar.exists() || !minecraftCommonJar.exists();
+		boolean requiresRefresh = isRefreshDeps() || !Files.exists(minecraftClientOnlyJar) || !Files.exists(minecraftCommonJar);
```

Comment:
```suggestion
		boolean requiresRefresh = isRefreshDeps() || Files.notExists(minecraftClientOnlyJar) || Files.notExists(minecraftCommonJar);
```

Reply:
Ah I didn’t know that was a thing. Idea could do with a hint for this. 👍

Reply:
Strictly speaking they're not complete opposites, in the case where it is not possible to determine both `exists` and `notExists` would return `false`. Doesn't matter in most cases though